README.md

```diff
@@ -152,6 +162,64 @@ And to run only the client tests, run the test:client task:
 $ grunt test:client
 ```
 
+
+## Running your application with Gulp
+
+After the install process, you can easily run your project with:
+
+```bash
+$ gulp
+```
+or
+
+```bash
+$ gulp default
+```
+
+The server is now running on http://localhost:3000 if you are using the default settings. 
+
+### Errors at Runtime
```

Comment:
I will remove that section.   Will look into a/the troubleshooting guide.  I apologize, still catching the ropes of contribution.  I'll make the changes and recommit after work today.  I'm hoping to make it easier for people to get started with MEAN.js and those were some of the roadblocks I encountered.